src/SectionLede/SectionLede.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, ReactNode } from 'react';
+import { color, spacing } from '../_tokens';
+import { minLg, minMd } from '../_helpers';
+
+import { Text } from '../Text/Text';
+import { styled } from '@storybook/theming';
+
+const ContentContainer = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+`;
+
+const ContentWrapper = styled.div`
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+
+  & > * + * {
```

Comment:
You could potentially use the stack component for this https://642d765a7e8afcfb104268dc-rwvrrcnvkj.chromatic.com/?path=/docs/layout-stack--docs but this works too

Reply:
Good call, thanks. I updated it for consistency.